Paccy10/olympus-api
# Feature(users): get user profile by username

**What does this PR do?**

Adds get user profile functionality

**Description of Task to be completed?**

You can get any user profile by username

**How should this be manually tested?**

- Checkout to the branch `ft-get-user-profile`
- Run the app with flask run
- Go to postman and send a `GET` request to `/users/profile/{username}` and replace `username` with the user username
- The user profile should be fetched

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

N/A

**Screenshots (if appropriate)**

N/A

**Questions:**

N/A
